src/logger.rs

```diff
@@ -1,25 +1,21 @@
-use std::env;
-
-use once_cell::sync::Lazy;
-
-static GHA: Lazy<bool> = Lazy::new(|| env::var_os("GITHUB_ACTIONS").is_some());
+use crate::github;
 
 pub fn error_exit(message: &str) -> ! {
     error(message);
     std::process::exit(1);
 }
 
 pub fn error(message: &str) {
-    if *GHA {
-        println!("::error file=website-stalker.yaml::{message}");
+    if *github::IS_RUN_AS_GITHUB_ACTION {
+        github::error(message);
```

Comment:
Very unlikely thought experiment:
If we end up executing this code outside of GH Action but in my shell, I would see "::error foo bar ..". Would it make sense to prefix it to indicate "Website Stalker assumes GH Action, here is an error based on that assumed environment"? To give a stupid example: The failure "GITHUB_SERVER_URL is not set" makes sense in the Action but not in my shell.